freeswitch.spec

```diff
@@ -122,12 +122,10 @@ Vendor:       	http://www.freeswitch.org/
 #
 ######################################################################################################################
 Source0:        http://files.freeswitch.org/%{name}-%{nonparsedversion}.tar.bz2
-Source1:	http://files.freeswitch.org/downloads/libs/v8-3.24.14.tar.bz2
-Source2:	http://files.freeswitch.org/downloads/libs/mongo-c-driver-1.1.0.tar.gz
```

Comment:
@sergey-safarov  Source2 is missing (Source0, Source1, Source3, ...)

Reply:
That is not important.
In future, we can use Soruce2 for another file.

Reply:
It looks bad, please move Source5 to Source2

Reply:
PR updated

Reply:
@sergey-safarov Thanks